### PR TITLE
Semantic html

### PR DESCRIPTION
**Pull request by FusionForScience**
My branch name is a typo :(

<h1> An HTML/CSS rewrite: </h1>

- It uses [semantic tags](https://internetingishard.com/html-and-css/semantic-html/)
- Cleaner HTML and CSS code
- CSS that adheres a bit better to DRY
- Move CSS out of HTML
- Replace table use with flexbox use
- More modular CSS
- No more `<br><br>` - uses CSS margins

This sets the stage for a <strong> mobile layout</strong> as well as being better practice.

### CODE REVIEW

**Comment by FusionForScience** [comment]
For the team picture thing, we can keep how it looks on the desktop and use a media query to change the layout for mobile, hence the effort in this branch to modularize the code.

File: source/templates/home.pug [hunk 1]
```diff
@@ -1,75 +1,69 @@
 extend default
 
 block content
-  div(style={"min-height": "38vh", "display": "flex", "align-items": "center"})
-    .block
-      table
-        tr
-          td(style={"vertical-align": "middle", "width": "50%"})
-            img(src="/images/squad.jpg", style={"width": "100%"})
+  section(class="content-section" id="intro-section")
```
Comment (amikhalev) [inline source/templates/home.pug]: one big feature of pug is that you can write stuff like this much shorter as such:
```pug
section.content-section#intro-section
```

**Comment by amikhalev** [comment]
> For the team picture thing, we can keep how it looks on the desktop and use a media query to change the layout for mobile, hence the effort in this branch to modularize the code.

@FusionForScience 
I think that a media query would not be necessary for something like that. Just making the surrounding container a flexbox with `flex-direction: row` and `flex-wrap: wrap` and setting a min-width on the image should be enough to make it move when the page is shrunk. In general with my changes so far I've been avoiding media queries in favor of that.

I'm not sure if you need to do that anyways because I will be working on adding an image carousel to the home page shortly anyways, so as far as I can tell that will be replacing/absorbing spinny with the squad

File: source/templates/home.pug (hunk 1 above)
Comment (FusionForScience) [reply source/templates/home.pug]: For sure, I went ahead and changed that

File: source/stylesheets/index.styl [hunk 2]
```diff
@@ -149,6 +151,60 @@ h1
   padding          8px
   overflow-x       auto
 
+  h1
+    margin-bottom: 30px
+
+// This is the bold part of the header
+// ex. *We are UAS@UCLA,* ...
+.bold-header
+  font-weight: 900
```
Comment (DumboOctopus) [inline source/stylesheets/index.styl]: Do we have a readability rule on whether or not we put a colon between css attributes and their values?

Comment (comran) [reply source/stylesheets/index.styl]: Stylus doesn't require it. I've been a fan of no colons and indenting all attribute names/values at set indent levels for readability (2 character indent for name, 32 character indent for value). I wouldn't be opposed to another method though, so long as we agree on something.
indents

**Pull request merged**